### ISSUE TITLE
cannot set hotkey for phrase/script

### ISSUE THREAD

**Issue by GoogleCodeExporter** [eadevlin/autokey#23]
```
Steps to reproduce the problem
1. Open configuration dialog
2. Select any phrase or script
3. Press "Set" button for hotkey
4. Press "Press to set"
5. Press "Cancel"
6. Press "Set" button for hotkey again

I would expect that the "Press to set" button is enabled but it's still 
disabled.  So I cannot set a new hotkey for this phrase anymore.

Version:
Autokey 0.61.2 (KDE 4.4.0) under Ubuntu 10.4.

```

Original issue reported on code.google.com by `ste...@endrullis.de` on 28 Feb 2010 at 8:05

**Comment by GoogleCodeExporter** [comment]
```
Thank you for reporting
```

Original comment by `cdekter` on 24 Mar 2010 at 11:44
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in 0.61.6
```

Original comment by `cdekter` on 25 Mar 2010 at 1:29
- Changed state: **Fixed**
